crypto/fipsmodule/rsa/rsa_impl.c

```diff
@@ -105,6 +105,13 @@ int rsa_check_public_key(const RSA *rsa, rsa_asn1_key_encoding_t key_enc_type) {
   // Slighty more thorough validation of |n > e|/
   assert(BN_ucmp(rsa->n, rsa->e) > 0);
 
+  // RSA moduli must be odd. In addition to being necessary for RSA in general,
+  // we cannot setup Montgomery reduction with even moduli.
+  if (!BN_is_odd(rsa->n)) {
+    OPENSSL_PUT_ERROR(RSA, RSA_R_BAD_RSA_PARAMETERS);
+    return 0;
+  }
+
```

Comment:
What do you think about moving this up above the Java/ACCP fix? I resolved this conflict and moved it up so the return 1 on line 102 doesn't allow an even key to be imported.

Reply:
Sure I thinking moving it up would make sense since stripped keys still check for n